### ISSUE TITLE
Pass variables other than field values

### ISSUE THREAD

**Issue by ThisIsMyFavouriteJar** [whitecube/nova-flexible-content#316]
Hello,

I love the package, and have been playing around with it for a while. I have a custom layout, and a custom resolver, and was wondering if it is possible to pass variables other than field values to my layout.

In short I have this in my Resolver, basically I want to fill my Tags autocompleteItems (in Layout) with a value passed through from the Resolver, as shown below.  

```
........
    public function get($resource, $attribute, $layouts)
    {
            return $layout->duplicateAndHydrate($question->id, [
                'id' => $question->id,
                'name' => $question->name,
                'meta' => $resource->meta,
            ]);
        })->filter();
    }
.......
```

And this in my Layout:

```
    public function fields()
    {
        return [
   
            Hidden::make('ID', 'id'),
            Text::make('Name')->rules('required', 'max:200'),
            Tags::make('Variables', 'variables')
                    ->autocompleteItems($this->meta ?? [])
                    ->nullable(),
           
        ];
    }
```

Is this possible? Any help about this would be greatly appreciated! 
I had a look at Flexible Content, but I am not sure if that fits my usecase, 
because as I understood it it is more for additional Layouts.

Again, all help appreciated!

Cheers

**Comment by toonvandenbos** [comment]
Did you find a solution? Since it is not really an issue with the package I'll close this issue but your use case seems interesting.

**Comment by ThisIsMyFavouriteJar** [comment]
Hi Toon,

This is how I got this working for now. At least it is meeting my expectation, perhaps it will help somebody else:

From my Nova Resource I pass information like this (e.g. parent information)

```
Flexible::make('Children')
                    ->preset(\App\Nova\Flexible\Presets\ChildrenPreset::class, ['parent' => $this->toArray()])
```

And retrieve it like this in, in my case in ChildrenPreset :

```
public function __construct($parent)
    {
        $this->parent = $parent;
    }
```

Now I can access the variables as usual, using $this-> .
(At least before I updated to the latest version ;) )

Succes ermee en bedankt voor de package!
